File: 707_Tests/MainFormStarter/MainFormStarter/MainForm.vb
Imports System
Imports System.Drawing
Imports System.Windows.Forms


Public NotInheritable Class MainForm
    Public Sub New()
        ' This call is required by the designer.
        InitializeComponent()

        ' Add any initialization after the InitializeComponent() call.
        Me.p_InitOnce()
    End Sub

    ' ::
    Private Sub p_InitOnce()
        Dim t_s As Size = Me.ClientSize
        t_s.Width = 800
        t_s.Height = 600
        Me.ClientSize = t_s
        Me.StartPosition = FormStartPosition.CenterScreen
        Me.Text = "BJ용연향 멘트 관리툴 Ver 1.21"

    End Sub

    ' ::
    Private Sub p_This_Load(ByVal sender As Object, ByVal ea As EventArgs) Handles MyBase.Load

    End Sub

End Class

File: 707_Tests/CaseByLogical_001/CaseByLogical_001/MainForm.cs
using System;
using System.Windows.Forms;


namespace CaseByLogical_001
{
    public sealed partial class MainForm : Form
    {
        // ::
        public MainForm()
        {
            InitializeComponent();

            this.p_initOnce();
        }

        // ::
        private void p_Load(object sender, EventArgs ea)
        {
            this.Text = "CaseByLogical Ver 1.01";

            this._richTextBox1.MaxLength = 99999;
        }

        // ::
        private void p_initOnce()
        {

        }
    }
}
